SpatialGDK/Documentation/content/technical-overview/gdk-concepts.md

```diff
@@ -14,32 +14,32 @@ Key concepts in Unreal Engine map to concepts in the GDK, as shown in the table
 
 ## GDK for Unreal concepts
 We’ve introduced some new concepts to facilitate the fact that SpatialOS enables you to spread computation between multiple servers - known as “server-worker instances” in SpatialOS.
 
 ### Offloading
-Offloading is the new architecture that the SpatialOS GDK for Unreal provides to allocate the authority of specific Actor groups from the main Unreal server worker instance to a different worker instance. By using offloading, you can save the resources of the main Unreal server-worker instance when you want to build richer game features.
+Offloading is one of the multiserver options for working with SpatialOS. It involves allocating the authority over specific groups of Actors to a server-worker instance other than your main Unreal server-worker instance. 
+
+For example, you could offload computationally expensive but latency-tolerant systems, such as advanced AI or large-scale background physics simulation, so that your main server-worker instance can run other systems at a larger scale.
 
 <%(Lightbox image="{{assetRoot}}assets/offloading-diagram.png")%>
-_Offloading: Offloaded Unreal server-worker instance has authority only over Red Actors and the Main Unreal server-worker instance that runs major game systems has authority over all Actors except the Red Actors._
+_Offloading: The offloaded server-worker instance has authority only over Red Actors, and the main Unreal server-worker instance that runs major game systems has authority over all other Actors._
 
 For more information, see [Offloading overview]({{urlRoot}}/content/workers/offloading-concept).
 
 ### Actor groups
-To facilitate offloading, we've created the concept of Actor groups to help you configure what Actor types that a given server-worker type has authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
+To facilitate offloading, we've created the concept of Actor groups to help you configure the Actor types that a given type of server-worker instance has authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
 
-Before you start to use offloading in your game, ensure that you’re familiar with the [best practices for using offloading]({{urlRoot}}/content/workers/offloading-concept#best-practices) and [offloading workflow]({{urlRoot}}/content/workers/set-up-offloading).
+Before you start to use offloading in your game, make sure that you’re familiar with the [best practices]({{urlRoot}}/content/workers/offloading-concept#best-practices) and with how to [set up your game feature for offloading]({{urlRoot}}/content/workers/set-up-offloading).
```

Comment:
This is a bit bonkers, as no-one's going to try offloading without actually finding out about it first, or are they?
We usually say "for guidance" or "for more information".
```suggestion
For guidance on offloading, see the documentation on offloading [best practices]({{urlRoot}}/content/workers/offloading-concept#best-practices) and [how to set up your game for offloading]({{urlRoot}}/content/workers/set-up-offloading).
```
Totally gonna rename that title so renaming it here.

Reply:
>This is a bit bonkers, as no-one's going to try offloading without actually finding out about it first, or are they?
See how I thought, Naaah, no-one would and then thought - maybe they are!